windows/src/main/csharp/ch/cyberduck/ui/ViewModels/TransfersViewModel.cs

```diff
@@ -608,6 +615,7 @@ private void ValidateCommands()
         StopCommand.NotifyCanExecuteChanged();
         TrashCommand.NotifyCanExecuteChanged();
 
+        // TODO Reduce triggering this a billion times a second
```

Comment:
@AliveDevil Resolve TODO

Reply:
Not now. Would require reevaluating how all commands/notifications are handled in this Window. Future improvement.